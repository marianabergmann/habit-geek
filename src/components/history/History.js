import React from "react";

const History = () => {
  return (
    <div className="large-size-container">
      <h2 className="mb-3">Here is your progress so far:</h2>

      <div className="mb-3"></div>
    </div>
  );
};

export default History;
