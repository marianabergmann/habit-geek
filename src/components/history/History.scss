.history {
  &__header {
    padding: 0;
  }

  &__header-button {
    width: 100%;
  }
}
