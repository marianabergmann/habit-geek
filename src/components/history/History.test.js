import React from "react";
import { shallow } from "enzyme";
import History from "./History";

describe("History component", () => {
  it("Renders static elements", () => {
    const history = shallow(<History />);
    expect(history).toMatchSnapshot();
  });
});
