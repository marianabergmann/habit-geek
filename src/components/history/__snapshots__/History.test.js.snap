// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component Renders static elements 1`] = `
<div
  className="large-size-container"
>
  <h2
    className="mb-3"
  >
    Here is your progress so far:
  </h2>
  <div
    className="mb-3"
  />
</div>
`;
